src/Illuminate/Validation/Rules/AnyOf.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Illuminate\Validation\Rules;
+
+use Illuminate\Contracts\Validation\Rule;
+use Illuminate\Support\Facades\Validator;
+use Illuminate\Support\Str;
+
+class AnyOf implements Rule
```

Comment:
The Rule contract is deprecated. Do you mind of implementing `Illuminate\Contracts\Validation\ValidationRule` instead? Sorry for not mentioning this at first.